### DISCUSSION TITLE
I am very disapointed

### DISCUSSION THREAD

**Discussion by thminuth** [primefaces/community#3160] [PrimeNG]
I spent money on templates and blocks and figma designs because I thought that with prime and primeng I would have a good basis for development and be future-proof. Not to mention the time it takes to get to grips with the framework. Now, after 1 1/2 years, I'm just disappointed and annoyed. The blocks and templates were never updated. Errors are only fixed very slowly. The switch to Angular 18 and primeng 18 is a disaster. No changes to the beta status since September. I'm fed up and am switching to ngzorro.

**Comment by mertsincan** [comment]
I'm sorry to hear this, but we have been releasing RC versions regularly and are planning to release version 18.Final this week. In addition, we changed blocks structure. After v18.Final, we will adapt templates and blocks to our new theming structure. Also, we have completely updated Figma UI kit.

Versions;
https://www.npmjs.com/package/primeng/v/18.0.0-rc.2
...

Blocks;
https://primeblocks.org/

Figma Links; 
https://v18.primeng.org/uikit
https://www.figma.com/design/G855HjuSyK8viJr0a5ZjRG/Preview-%7C%C2%A0PrimeOne-%7C-3.0.1?node-id=830-41631
...

And;
https://github.com/primefaces/primeng/issues/16827#issuecomment-2511150789
